applications/plugins/CGALPlugin/MeshGenerationFromImage.h

```diff
@@ -146,7 +155,7 @@ class MeshGenerationFromImage : public sofa::core::DataEngine
     sofa::core::objectmodel::Data<bool> frozen;
 
     //Parameters
-    sofa::core::objectmodel::Data<double> facetAngle, facetSize, facetApproximation;
+    sofa::core::objectmodel::Data<double> edgeSize, facetAngle, facetSize, facetApproximation;
```

Comment:
edgeSize should follow the d_edgeSize naming convention...but for consistency let's say it is not a problem here :)